catalyst_voices/packages/catalyst_voices_services/lib/src/user/user_service.dart

```diff
@@ -0,0 +1,223 @@
+import 'dart:async';
+
+import 'package:catalyst_cardano_serialization/catalyst_cardano_serialization.dart';
+import 'package:catalyst_voices_models/catalyst_voices_models.dart';
+import 'package:catalyst_voices_services/catalyst_voices_services.dart';
+import 'package:logging/logging.dart';
+
+abstract interface class UserService {
+  Account? get account;
+
+  Stream<Account?> get watchAccount;
+
+  Keychain? get keychain;
+
+  Future<List<Keychain>> get keychains;
+
+  Stream<Keychain?> get watchKeychain;
+
+  Future<void> useActiveAccount();
```

Comment:
restoreActiveAccount? `use` sounds as if this code `changed` something while it only restores what was there on the last session

Reply:
we already have `restore` account function. I don't want to confuse those two.

maybe `useLastAccount` will be better?

Reply:
Fine, thanks, lets use `useLastAccount`